src/api/resources/airport.py

```diff
@@ -8,3 +8,11 @@ class Airport(BaseModel):
     runways: list[Runway]
     total_arrivals: int
     total_departures: int
+
+    def to_json(self) -> dict[str, str | int | list[dict[str, int]]]:
```

Comment:
Can you rename this to `json` to be consistent with the other methods in other types please?